packages/babel-plugin/__tests__/stylex-transform-call-test.js

```diff
@@ -1910,5 +1910,28 @@ describe('@stylexjs/babel-plugin', () => {
         }[!!(sidebar == null && !isSidebar) << 2 | !!isSidebar << 1 | !!isContent << 0];"
       `);
     });
+
+    test('Using styles from a for loop', () => {
+      expect(
+        transform(
+          `
+          import stylex from '@stylexjs/stylex';
+          function test(colors, obj) {
+            for (const color of colors) {
+              obj[color.key] = stylex(color.style);
```

Comment:
Is there already a test for this against the `stylex.props` syntax?

Reply:
I'll add it. For now I was testing the specific code that caused a breakage.